projects/01-pair-matching/hanie-abutalebi/src/main.js

```diff
@@ -0,0 +1,115 @@
+import './style.scss';
+
+const selectClass = {
+  board: document.querySelector('.board'),
+  board_box: document.querySelector('.board-box'),
+  win: document.querySelector('.win'),
+};
+let flippedCards = 0;
+let dimNumber;
+const shuffle = (array) => {
+  const newArray = [...array];
+  for (let index = newArray.length - 1; index > 0; index -= 1) {
+    const randomIndex = Math.floor(Math.random() * (index + 1));
+    const original = newArray[index];
+    newArray[index] = newArray[randomIndex];
+    newArray[randomIndex] = original;
+  }
+  return newArray;
+};
+const selectRandom = (array, items) => {
+  const newArray = [...array];
+  const randomPicks = [];
+
+  for (let index = 0; index < items; index += 1) {
+    const randomIndex = Math.floor(Math.random() * newArray.length);
+    randomPicks.push(newArray[randomIndex]);
+    newArray.splice(randomIndex, 1);
+  }
+  return randomPicks;
+};
+
+const generateGame = () => {
```

Comment:
Anywhere you called `generateGame` and `setEventListeners` (like `generateGame(dimNumber);` in line 104 and some lines after that) , you passed them an argument (like `dimNumber`), but you're not using the argument in their declarations!
Please fix this.